scripts/sct_straighten_spinalcord.py

```diff
@@ -861,13 +874,19 @@ def get_parser():
                       type_value=None,
                       description="Disable curved to straight transformation computation.",
                       mandatory=False)
-    parser.add_option(name="-resample",
+    parser.add_option(name="-speed_factor",
                       type_value='float',
-                      description='Isotropic resolution of the straightening output, in millimeters.\n'
-                                  'Resampling to lower resolution decreases computational time while decreasing straightening accuracy.\n'
-                                  'To keep native resolution, set this option to 0.\n',
+                      description='Acceleration factor for the calculation of the straightening warping field.\n'
+                                  'This speed factor enables an intermediate resampling to a lower resolution, which decreases the computational time while decreasing straightening accuracy.\n'
+                                  'A speed factor of 2 means that a downsampling of the input image by a factor 2 will be applied to calculate the straightening warping field. For example, a 1x1x1 mm^3 image will be downsampled to 2x2x2 mm3, providing a speed factor of approximately 8.\n'
+                                  'Note that accelerating the straightening process reduces the precision of the algorithm, and induces undesirable edges effects. To keep the native resolution, set this option to 0.\n',
                       mandatory=False,
                       default_value=0)
+    parser.add_option(name="-xy_size",
+                      type_value='float',
+                      description='Change the size of the XY FOV, in mm. The resolution is the same as the source image (-i).\n',
```

Comment:
this description is unclear. Does that refer to the destination image? if so, i would rewrite as: "Set the size of the XY FOV of the destination image, in mm.